views/widget.subscribe.display.php

```diff
@@ -363,4 +366,17 @@
 				</span>
 		</form>
 	</div>
+	<?php
+		if ( isset( $sailthru['google_recaptcha_site_key'] ) && ! empty( $sailthru['google_recaptcha_site_key'] ) ) {
```

Comment:
I don't necessarily like this but it works. Are there any better alternatives?

Reply:
Why is one `isset` and one `!empty`? But also, I'd say you can turn it into a boolean function.

e.g. 
```php
if ( has_recaptcha_settings( $sailthru ) ) {
     ...
}
```

Reply:
could actually do without `isset` but `empty` catches for empty strings, arrays, NULL, etc

Reply:
Ah, thought one was checking for the secret key. Should you also be checking to verify there's a secret key?

re JS, there's a few ways to do that. I think you can just literally do a multiline string if you'd like, but there's also this: https://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.heredoc

We can also wait for WordPress feedback on this one.